### PR TITLE
EDU-965: tag CLI source nodes

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR adds additional tags to CLI source nodes.

## Notes to reviewers
Branch had better be cleared up.

### CODE REVIEW

File: docs-src/cli/activity/index.md [hunk 1]
```diff
@@ -3,10 +3,20 @@ id: index
 title: temporal activity index
 sidebar_label: activity
 description: Operations that can be performed on Workflow Activities.
+<<<<<<< HEAD
```
Comment (flossypurse) [inline docs-src/cli/activity/index.md:6]: merge issue

**Review by flossypurse — CHANGES_REQUESTED** [review]

File: docs-src/cloud/tcld/account/get.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sidebar_label: get
 description: How to get information about an account in Temporal Cloud using tcld.
 tags:
   - tcld
-  - cli reference
+  - cli-reference
```
Comment (flossypurse) [inline docs-src/cloud/tcld/account/get.md:8]: make sure these are `cli reference` - no space

**Review by flossypurse — CHANGES_REQUESTED** [review]
Just revert the `cli-reference` change.

The tag needs to be `cli reference` - no space.

Otherwise, looking good.

**Comment by DJSanti** [comment]
@flossypurse Got the hyphens out. Just need your OK on this.

**Pull request merged**